parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoMessageConverter.java

```diff
@@ -54,13 +54,13 @@
   private final Message.Builder myBuilder;
 
   // used in record converter
-  ProtoMessageConverter(ParentValueContainer pvc, Class<? extends Message> protoClass, GroupType parquetSchema) {
-    this(pvc, Protobufs.getMessageBuilder(protoClass), parquetSchema);
+  ProtoMessageConverter(ParentValueContainer pvc, Class<? extends Message> protoClass, GroupType parquetSchema, boolean specCompliant) {
+    this(pvc, Protobufs.getMessageBuilder(protoClass), parquetSchema, specCompliant);
   }
```

Comment:
The parquet-protobuf should be able to read parquet spec compliant (3-level) repeated fields transparently, without relying a parameter/configuration.